CHANGELOG.md

```diff
@@ -1,4 +1,8 @@
-## 1.0.0  - 2023/05/01
+## 1.0.1 - 2023/02/02
+* 🚀 [FEATURE] Allow configuring custom array-like classes to be treated as collections when serializing.  More details can be found [here](https://github.com/blueprinter-ruby/blueprinter/pull/6). Thanks to [@toddnestor](https://github.com/toddnestor).
```

Comment:
felt weird thanking myself 😂 but wanted to be consistent with the changelog entries below.